packages/app-accounts/src/helpers/changePassword.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright 2017-2018 @polkadot/app-accounts authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import { TranslationFunction } from 'i18next';
+import { FormErrors } from '@polkadot/ui-app/types';
+
+const emptyPasswords = {
+  password: '',
+  newPassword: ''
+};
+
+const emptyErrors = () => {
+  return ({
+    formError: '',
+    inputError: emptyPasswords
+  } as FormErrors);
+};
+
+const emptyStateForModal = (showModal: boolean, address: string) => {
+  return ({
+    address,
+    error: emptyErrors(),
+    newPassword: '',
+    password: '',
+    showModal,
+    success: undefined
+  });
+};
+
+const errorForKey = (key: string, t: TranslationFunction) => {
+  const errors: any = {
+    'editor.change.password.input.password.differ.error': t('editor.change.password.input.password.differ.error', {
```

Comment:
We fall into this trap with each PR -

1. the keys are _only_ allowed inside the actual React component
2. lets refrain from trying to pass `t` in everywhere, rather keep it in the component and there only